HW1/P5/P5_helper.py

```diff
@@ -0,0 +1,82 @@
+from pyspark.accumulators import AccumulatorParam
+class BFSAccumulatorParam(AccumulatorParam):
```

Comment:
As in P4, this uses a large dataset for the accumulator, resulting in a non-negligible overhead.